resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -814,6 +814,326 @@
+            - label: NRAS Q61R
+              value: nras_q61r
+            - label: NRAS Q61K
+              value: nras_q61k
+            - label: NRAS G13R
+              value: nras_g13r
+            - label: NRAS G12D
+              value: nras_g12d
+            - label: NRAS Q61L
+              value: nras_q61l
+            - label: NRAS
+              value: nras
+        - name: mutation_status_atm
+          label: Mutation status/ATM
+          type: basic:string
+          required: false
+          choices:
+            - label: ATM R337C
+              value: atm_r337c
+            - label: 'ATM K2811Sfs*46'
+              value: 'atm_k2811sfs*46'
+            - label: 'ATM R250*'
+              value: 'atm_r250*'
+            - label: ATM
+              value: atm
+        - name: mutation_status_egfr
+          label: Mutation status/EGFR
+          type: basic:string
+          required: false
+          choices:
+            - label: EGFR L858R
+              value: egfr_l858r
+            - label: EGFR E746_A750del
+              value: egfr_e746_a750del
+            - label: EGFR G598V
+              value: egfr_g598v
+            - label: EGFR A289V
+              value: egfr_a289v
+            - label: EGFR
+              value: egfr
+        - name: mutation_status_braf
+          label: Mutation status/BRAF
+          type: basic:string
+          required: false
+          choices:
+            - label: BRAF V640E
+              value: braf_v640e
+            - label: BRAF V640M
+              value: braf_v640m
+            - label: BRAF
+              value: braf
+        - name: mutation_status_alk
+          label: Mutation status/ALK
+          type: basic:string
+          required: false
+          choices:
+            - label: ALK R1275Q
+              value: alk_r1275q
+            - label: ALK F1174L
+              value: alk_f1174l
+            - label: ALK
+              value: alk
+        - name: mutation_status_met
+          label: Mutation status/MET
+          type: basic:string
+          required: false
+          choices:
+            - label: MET
+              value: met
+        - name: mutation_status_ros1
+          label: Mutation status/ROS1
+          type: basic:string
+          required: false
+          choices:
+            - label: ROS1
+              value: ros1
+        - name: mutation_status_cdkn2a
+          label: Mutation status/CDKN2A
+          type: basic:string
+          required: false
+          choices:
+            - label: 'CDKN2A R80*'
+              value: 'cdkn2a_r80*'
+            - label: 'CDKN2A R58*'
+              value: 'cdkn2a_r58*'
+            - label: CDKN2A H83Y
+              value: cdkn2a_h83y
+            - label: CDKN2A
+              value: cdkn2a
+        - name: mutation_status_rbm10
+          label: Mutation status/RBM10
+          type: basic:string
+          required: false
+          choices:
+            - label: 'RBM10 F913='
```

Comment:
Synonymous variants are written like this in SnpEff `Phe913Phe`. Should this be synchronized?